config/rbac/base/kustomization.yaml

```diff
@@ -1,18 +1,10 @@
-# Adds namespace to all resources.
-namespace: gatekeeper-system
-
-# Value of this field is prepended to the
-# names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
-# Note that it should also match with the prefix (text before '-') of the namespace
-# field above.
-namePrefix: gatekeeper-operator-
-
-# Labels to add to all resources and selectors.
-commonLabels:
-  control-plane: gatekeeper-operator-controller-manager
```

Comment:
We still need this so that the resources in this directory contain this label instead of `control-plane: controller-manager`.